gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2520,6 +2334,215 @@ func preprocess1(store Store, ctx BlockNode, n Node) Node {
 	return nn
 }
 
+// defineOrDecl merges the code logic from op define (:=) and declare (var/const).
+func defineOrDecl(
+	store Store,
+	bn BlockNode,
+	isConst bool,
+	nameExprs []NameExpr,
+	typeExpr Expr,
+	valueExprs []Expr,
+) {
+	numNames := len(nameExprs)
+	numVals := len(valueExprs)
+
+	if numVals > 1 && numNames != numVals {
+		panic(fmt.Sprintf("assignment mismatch: %d variable(s) but %d value(s)", numNames, numVals))
+	}
+
+	if numNames < 1 {
+		panic("must have at least one name to assign")
+	}
+
+	sts := make([]Type, numNames) // static types
+	tvs := make([]TypedValue, numNames)
+
+	if numNames > 1 && len(valueExprs) == 1 {
+		parseMultipleAssignFromOneExpr(sts, tvs, store, bn, nameExprs, typeExpr, valueExprs[0])
+	} else {
+		parseAssignFromExprList(sts, tvs, store, bn, isConst, nameExprs, typeExpr, valueExprs)
+	}
+
+	node := bn
+	if fn, ok := bn.(*FileNode); ok {
+		node = fn.GetParentNode(nil).(*PackageNode)
+	}
```

Comment:
there is a helper for this: `skipFile(...)`

Reply:
Fixed